eng/common/pipelines/templates/steps/verify-changelog.yml

```diff
@@ -0,0 +1,44 @@
+parameters:
+  PackageName: 'not-specified'
+  ServiceName: 'not-specified'
+  ForRelease: ''
+
+steps:
+  - pwsh: |
+      Import-Module "$(Build.SourcesDirectory)/eng/common/scripts/modules/common-manifest.psd1"
```

Comment:
Lets put this into a script file so it is easy to run locally as well as in our pipeline

Reply:
We might want to put it as a function in our changelog script.

Reply:
I do still think it is a good idea have this code in a wrapper script so that we can easily run locally.